package.json

```
@@ -6,9 +6,11 @@
   "workspaces": [
     "src/core/*",
     "src/core/components/*",
-    "packages/@guardian/*"
+    "packages/@guardian/*",
+		"scripts/analysis"
   ],
   "scripts": {
+    "analysis": "ts-node ./scripts/analysis",
```

what do you htink of this?
```suggestion
    "get-usage": "ts-node ./scripts/analysis",
```
`analysis` doesn't quite explain what will happen to me

---

Yeah, I like it. Done in https://github.com/guardian/source/pull/972/commits/30305b6a42724d715c545e46c13e62450efff73a